src/routes/volume/detail/CreateRecurringJob.js

```diff
@@ -353,7 +353,7 @@ const modal = ({
                       required: true,
                     },
                   ],
-                })(<InputNumber disabled={noRetain(getFieldValue('task')) || isEdit} style={{ width: '80%' }} min={0} />)}
+                })(<InputNumber disabled={noRetain(getFieldValue('task'))} style={{ width: '80%' }} min={0} />)}
```

Comment:
Edit recurring job in volume detail page is using [CreateRecurringJob.js](https://github.com/a110605/longhorn-ui/blob/53c0ab092cc684d57644b1b38f1bf6ed96e63562/src/routes/volume/detail/RecurringJob.js#L6-L7)

```
import EditRecurringJob from '../../recurringJob/CreateRecurringJob'
```